examples/native/components/ControlExamples/Date/Date.tsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import { Text } from 'react-native';
+
+interface DateProps {
+  date: Date;
+}
+
+export const LocalisedDate = ({ date }: DateProps) => <Text>{date.toLocaleDateString()}</Text>;
```

Comment:
Could we also show the selected time here, not just the date?

```suggestion
export const LocalisedDate = ({ date }: DateProps) => (
  <Text>
    {date.toLocaleDateString()} {date.toLocaleTimeString()}
  </Text>
);
```